carAPI/src/main/java/com/everis/entity/Car.java

```diff
@@ -10,8 +10,10 @@
 import javax.persistence.NamedQuery;
 
 import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.SQLDelete;
 
 @Entity
+@SQLDelete(sql = "UPDATE car set deleted = true WHERE ID = ?")
```

Comment:
Why do you mix NamedQuery with SQLDelete, it should be possible to use only NamedQuery